src/components/button-favourite/button-favourite.tsx

```diff
@@ -1,23 +1,27 @@
 import React, { useCallback, useEffect, useState } from "react";
+import { useDispatch } from "react-redux";
```

Comment:
Please remove unused dependencies. This also applies elsewhere in this PR.

Reply:
👍